src/index.js

```diff
@@ -1,37 +1,57 @@
-import './index.css';
-
-const Lists = [
-  {
-    description: 'wash the dishes',
-    completed: true,
-    index: 0,
-  },
-  {
-    description: 'go to masjid',
-    completed: true,
-    index: 1,
-  },
-  {
-    description: 'Clear all completed',
-    completed: true,
-    index: 2,
-  },
-];
-
-const todoFullList = document.getElementById('todoFullList');
+import Add, { Remove } from './modules/funtions.js';
```

Comment:
- [ ] Well done implementing the changes required by the previous code reviewer, but you forgot to import the styles to match with the wireframe 👇🏽 

![image](https://user-images.githubusercontent.com/58642949/228810593-dd8e7159-d809-488f-a195-59cce38a2153.png)